### PR TITLE
mixer-telemetry chart should not depend on global.yaml

### PR DESCRIPTION
**Pull request by jacob-delgado**
Task for https://github.com/istio/istio/issues/25184

Find all occurrences of .Values.global in istio-discovery chart
  `rg -o -e ".Values.global(\.\w+)*" | awk -F\: '{print $2}' | sort -u`

Take values from global.yaml into values.yaml

[ ] Configuration Infrastructure
[ ] Docs
[X] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: manifests/charts/istio-telemetry/mixer-telemetry/values.yaml [hunk 1]
```diff
@@ -101,3 +101,190 @@ mixer:
...
+    s390x: 2
+    ppc64le: 2
+
+  # set the default set of namespaces to which services, service entries, virtual services, destination
+  # rules should be exported to. Currently only one value can be provided in this list. This value
+  # should be one of the following two options:
+  # * implies these objects are visible to all namespaces, enabling any sidecar to talk to any other sidecar.
+  # . implies these objects are visible to only to sidecars in the same namespace, or if imported as a Sidecar.egress.host
+  defaultConfigVisibilitySettings: []
+#  - '*'
```
Comment (ostromart) [inline manifests/charts/istio-telemetry/mixer-telemetry/values.yaml:122]: stray line?

File: manifests/charts/istio-telemetry/mixer-telemetry/values.yaml [hunk 2]
```diff
@@ -101,3 +101,190 @@ mixer:
...
+  # Default behavior: latest images will be Always else IfNotPresent.
+  imagePullPolicy: ""
+
+  # ImagePullSecrets for all ServiceAccount, list of secrets in the same namespace
+  # to use for pulling any images in pods that reference this ServiceAccount.
+  # For components that don't use ServiceAccounts (i.e. grafana, servicegraph, tracing)
+  # ImagePullSecrets will be added to the corresponding Deployment(StatefulSet) objects.
+  # Must be set for any cluster configured with private docker registry.
+  imagePullSecrets: []
+  # - private-registry-key
```
Comment (ostromart) [inline manifests/charts/istio-telemetry/mixer-telemetry/values.yaml:175]: stray line?

File: manifests/charts/istio-telemetry/mixer-telemetry/values.yaml [hunk 3]
```diff
@@ -101,3 +101,190 @@ mixer:
...
+  # The customized CA address to retrieve certificates for the pods in the cluster.
+  # CSR clients such as the Istio Agent and ingress gateways can use this to specify the CA endpoint.
+  caAddress: ""
+
+  # controlPlaneMtls enabled. Will result in delays starting the pods while secrets are
+  # propagated, not recommended for tests.
+  controlPlaneSecurityEnabled: true
+
+  # Used to locate istio-pilot.
+  # Default is to install pilot in a dedicated namespace, istio-pilot11. You can use multiple namespaces, but
```
Comment (ostromart) [inline manifests/charts/istio-telemetry/mixer-telemetry/values.yaml:223]: remove version specific info

File: manifests/charts/istio-telemetry/mixer-telemetry/values.yaml (hunk 2 above)
Comment (jacob-delgado) [reply manifests/charts/istio-telemetry/mixer-telemetry/values.yaml:175]: This was copied over from the `global.yaml` file. I'll remove it.

File: manifests/charts/istio-telemetry/mixer-telemetry/values.yaml (hunk 1 above)
Comment (jacob-delgado) [reply manifests/charts/istio-telemetry/mixer-telemetry/values.yaml:122]: It was copied over https://github.com/istio/istio/blob/master/manifests/charts/global.yaml#L307 but I'll remove it

File: manifests/charts/istio-telemetry/mixer-telemetry/values.yaml (hunk 3 above)
Comment (jacob-delgado) [reply manifests/charts/istio-telemetry/mixer-telemetry/values.yaml:223]: removed

**Review by ostromart — APPROVED** [review]

**Pull request merged**